portfolio/src/main/webapp/script.js

```diff
@@ -90,13 +90,25 @@ function cancelRedirect() {  // eslint-disable-line no-unused-vars
 
 /**
  * Obtains the stored comments and adds them to the comment
- * section on the home page.
+ * section on the home page. Removes the clear button and the
+ * option to selected number of comments displayed if there are
+ * no comments.
  */
 function getComments() {  // eslint-disable-line no-unused-vars
   const numComments = $('#num-comments-selector :selected').val();
   fetch('/data?max-comments=' + numComments)
       .then((response) => response.json())
       .then((messages) => {
+        if (messages.length == 0) {
```

Comment:
if (!messages || messages.length === 0)

messages might be null or undefined

Reply:
Oh, okay.